### PR TITLE
Add uSID endpoint enums for udx and udt

### PR DESCRIPTION
**Pull request by kperumalbfn**
Enums for uDX4, uDX6, uDT4, uDT6 and uDT46

### CODE REVIEW

**Comment by kperumalbfn** [comment]
@ashutosh-agrawal Please review

**Review by kcudnik — APPROVED** [review]

**Review by eddyk-nvidia — APPROVED** [review]

**Review by ashutosh-agrawal — APPROVED** [review]

**Review by kcudnik — APPROVED** [review]

File: inc/saisrv6.h [hunk 1]
```diff
@@ -102,6 +102,21 @@ typedef enum _sai_my_sid_entry_endpoint_behavior_t
...
+    /** Endpoint with decapsulation and IPv4 Cross-connect for uSID */
+    SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_UDX4,
+
+    /** Endpoint with decapsulation and IPv6 Cross-connect for uSID */
+    SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_UDX6,
+
+    /** Endpoint with decapsulation and specific IPv4 Table lookup for uSID */
+    SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_UDT4,
+
+    /** Endpoint with decapsulation and specific IPv4 Table lookup for uSID */
```
Comment (ahsalam) [inline inc/saisrv6.h:114]: There is a typo here it should be IPv6

**Review by ahsalam — APPROVED** [review]
The changes looks good. Thanks @kperumalbfn

**Review by cscarpitta — APPROVED** [review]

File: inc/saisrv6.h (hunk 1 above)
Comment (kperumalbfn) [reply inc/saisrv6.h]: Thanks, updated.

**Comment by kperumalbfn** [comment]
Closing this PR due to sanity check issue and force push failed. Opened https://github.com/opencomputeproject/SAI/pull/2102

**Pull request closed without merging**